protocol/communities/manager.go

```diff
@@ -63,8 +65,16 @@ func NewManager(identity *ecdsa.PublicKey, db *sql.DB, logger *zap.Logger, verif
 		}
 	}
 
+	lc := zap.NewDevelopmentConfig()
+	lc.OutputPaths = []string{"stdout"}
+	archiveLogger, err := lc.Build()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create a archive logger")
+	}
```

Comment:
I've added a new logger so we get some output in stdout when archives are created/downloaded (easier for debugging, kinda hard when stuff only gets written to geth.log)